### ISSUE TITLE
Bot crashes when Telegram bot API is unavailable

### ISSUE THREAD

**Issue by jyap808** [rockneurotiko/go-tgbot#2]
Error:

```
panic: Some error happened, maybe your token is bad:
Error code: 403
Description: 
Token: [MY API TOKEN]

goroutine 1 [running]:
github.com/rockneurotiko/go-tgbot.NewTgBot(0x4abbc0, 0x2c, 0x5)
    [MY GO PATH]/src/github.com/rockneurotiko/go-tgbot/tgbot.go:42 +0x3f5
main.main()
    [MY GO PATH]/src/github.com/jyap808/my-tgbot/helloworldecho.go:15 +0x2c

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
    /usr/local/Cellar/go/1.5/libexec/src/runtime/asm_amd64.s:1696 +0x1
exit status 2
```

**Comment by rockneurotiko** [comment]
@jyap808 Yeah, I hitted this issue too, I'm going to add a new constructor (NewWithError) that return (*TgBot, error) so we all can avoid this problem :)

**Comment by rockneurotiko** [comment]
Fixed with 85e6f47
